### PR TITLE
fix 'certficate' typo, regen files

### PR DESCRIPTION
**Pull request by dmitris**
#### Summary
fixed a minor typo (`certficate` with missing `i`), regenerated the files

#### Release Note
NONE

#### Documentation
not required

### CODE REVIEW

File: pkg/generated/restapi/operations/timestamp_server_api.go [hunk 1]
```diff
@@ -350,6 +350,6 @@ func (o *TimestampServerAPI) AddMiddlewareFor(method, path string, builder middl
 	}
 	o.Init()
 	if h, ok := o.handlers[um][path]; ok {
-		o.handlers[method][path] = builder(h)
+		o.handlers[um][path] = builder(h)
```
Comment (bobcallaway) [inline pkg/generated/restapi/operations/timestamp_server_api.go:353]: this is an unexpected change, i wonder if we don't have the swagger tool pinned?

Comment (haydentherapper) [reply pkg/generated/restapi/operations/timestamp_server_api.go:353]: Should be - https://github.com/sigstore/timestamp-authority/blob/main/hack/tools/go.mod#L6

Comment (haydentherapper) [reply pkg/generated/restapi/operations/timestamp_server_api.go:353]: Did you run `make` to generate this?

Comment (haydentherapper) [reply pkg/generated/restapi/operations/timestamp_server_api.go:353]: My guess is it's from updating https://github.com/sigstore/timestamp-authority/pull/386, and we don't check that no files would be changed.

**Review by haydentherapper — APPROVED** [review]

**Review by cpanato — APPROVED** [review]

**Pull request merged**